packages/client/src/forms/register/fieldMappings/birth/mutation/registration-mappings.ts

```diff
@@ -49,26 +48,27 @@ export function transformCertificateData(
         name: [
           {
             use: 'en',
-            firstNames: certificateData.collector.firstName,
-            familyName: certificateData.collector.lastName
+            firstNames: certificateData.collector.firstName as string,
+            familyName: certificateData.collector.lastName as string
```

Comment:
In a future refactor, we could get rid of `IFormFieldValue` some way.. it's almost comparable to `any`. Would help us get rid of these "as foo". 
```
export type IFormFieldValue =
  | string
  | string[]
  | number
  | boolean
  | Date
  | IDeclarationCertificate
  | IFileValue
  | IAttachmentValue
  | FieldValueArray
  | FieldValueMap
  | IContactPoint
  | IInformant
  | IDateRangePickerValue
```